packages/project-access/src/project/search.ts

```diff
@@ -499,3 +499,27 @@ export async function findCapProjects(options: {
     }
     return Array.from(result);
 }
+
+/**
+ * Returns true if the specified target path contains a CAP project.
+ * Checks sub and root directories.
+ *
+ * @param targetDir the target directory path.
+ * @returns true if CAP Project is in the directory or false if not.
+ */
+export async function checkPathForCapApp(targetDir: string): Promise<string | boolean> {
```

Comment:
We could updated (simplified) to be 

```
export async function checkPathForCapApp(targetDir: string): Promise<boolean> {
    // Check if targetDir is a CAP project
    const capType = await getCapProjectType(targetDir);

    // If not a CAP project root, check if the CAP root is in the subdirectories
    return !!capType || !!(await findCapProjectRoot(targetDir));
}
```

Reply:
The comments can probably be removed, the JSDoc can explain the process

Reply:
Thank you for the review.
I agree, however, multiple colleagues were asking about the need for the two functions to do a similar thing so I might leave the comments if that is okay?

Reply:
Ok, sure